TAO/tao/Transport.cpp

```diff
@@ -164,6 +163,8 @@ TAO_Transport::TAO_Transport (CORBA::ULong tag,
 #endif /* TAO_HAS_TRANSPORT_CURRENT == 1 */
   , flush_in_post_open_ (false)
 {
+  cache_map_entry_.entry_ = 0;
+  cache_map_entry_.int_id_ = 0;
```

Comment:
Transport::cache_map_entry_ needs to be POD, for qsort used to sort those elements in Transport_Cache_Manager_T to be able to memcpy its memory. This is the reason I avoided temptation to add any functions, including the default constructor.